### ISSUE TITLE
[spec] Question / possible limitation? Applying lookups per-script

### ISSUE THREAD

**Issue by jamesgk** [adobe-type-tools/afdko#145]
I was recently trying to come up with a way to apply a feature to some scripts but not others, essentially ignoring language, but couldn't think of a good way to do so with feature syntax. I've found that just the following source:

```
feature liga {
    script latn;
    sub f i by fi;
} liga;
```

doesn't even seem to exclude the "fi" ligatures from non-latn scripts. For example, testing output with harfbuzz:

```
$ hb-shape --script=arab out.otf "afib"
[b=3+1150|fi=1+1135|a=0+1114]
```

Another issue is that the following source:

```
feature locl {
    script latn;
    language FRA;
    sub [uni2009] by [uni2009.fra];
} locl;

feature liga {
   script latn;
   sub f i by fi;
} liga;
```

excludes the "fi" ligation from French, since a languagesystem is made for latn/FRA to include uni2009 -> uni2009.fra but the ligation is only included in the latn/dftl languagesystem.

So basically I'm wondering a) why my first example happens at all, and b) how I can achieve per-script inclusion of substitution features.

**Comment by jamesgk** [comment]
cc @behdad @brawer

**Comment by jamesgk** [comment]
Never mind, resolved this. a) happens because neither Arabic nor DFLT is in the table, so harfbuzz defaults to Latin. For b) one can simply list every language used in the feature source separately under the script, other languages will be covered by the default table. Not the cleanest, but it works.
